frappe/public/scss/login.bundle.scss

```diff
@@ -4,6 +4,8 @@ body {
 	@include media-breakpoint-up(sm) {
 		background-color: var(--bg-light-gray);
 	}
+	display: flex;
+	flex-direction: column;
```

Comment:
it should be in the body of all website pages not only on the login page.
in base.scss

Reply:
Avoided doing this because lots of website pages use it and top level flex like this might break them 😅

Reply:
Let's do in develop I don't think it will break anything